pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -70,6 +70,7 @@ through Admin Router.
 | `dcos:adminrouter:acs`<br>Controls access to the security and access management features.                                                                                                                                                                                         | x    |   |   |   |   |
 | `dcos:adminrouter:ops:ca:ro`<br>Controls access to the read-only endpoints of the [Certificate Authority API](/1.10/security/ent/tls-ssl/ca-api/) and the `dcos security cluster ca` commands of the [DC/OS Enterprise CLI](/1.10/cli/enterprise-cli/).                               | x    |   |   |   |   |
 | `dcos:adminrouter:ops:ca:rw`<br>Controls user access to all endpoints of the [Certificate Authority API](/1.10/security/ent/tls-ssl/ca-api/) and the `dcos security cluster ca` commands of the [DC/OS Enterprise CLI](/1.10/cli/enterprise-cli/).                                    | x    |   |   |   |   |
+| `dcos:adminrouter:ops:cockroachdb`<br> Controls access to the [CockroachDB UI](https://www.cockroachlabs.com/docs/v2.0/admin-ui-overview-dashboard.html).      | x    |   |   |   |   |
```

Comment:
This links to CockroachDB 2.0 but I believe that we are using an earlier version.

Reply:
:100: